lib/sqlalchemy/dialects/oracle/types.py

```diff
@@ -214,6 +226,21 @@ def as_generic(self, allow_nulltype=False):
             day_precision=self.day_precision,
         )
 
+    @property
+    def python_type(self) -> Type[dt.timedelta]:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

why this decorator?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4873

Reply:
**mike bayer** (zzzeek) wrote:

somehow i just didnt see that.  will remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4873